plugins/lookup/__init__.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Brian Scholer (@briantist)
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible.errors import AnsibleError
+# from ansible.plugins import AnsiblePlugin
```

Comment:
It's probably a good idea to remove the out-commented imports before merging this, as these tend to stay around and confuse for a long time ;-)

Reply:
agreed, right now they are reminders for me as I work on it :)